### PR TITLE
Improve side panel icons

### PR DESCRIPTION
**Pull request by kanchanagodage**
- pencil-icon
- key-icon

### Preview
admin-portal-icon-update
pencil-key-icon-preview
user-stores-illustrations

### CODE REVIEW

File: modules/theme/src/themes/default/assets/images/icons/gears-icon.svg [hunk 1]
```diff
@@ -20,8 +20,83 @@
...
-    <path class="path primary" d="M483.453,96.66a80.861,80.861,0,0,1,1,10,80.924,80.924,0,0,1-1,10.008l19.223,14.547a10.68,10.68,0,0,1,2.793,13.848l-17.258,29.867a10.677,10.677,0,0,1-13.379,4.5l-22.121-9.324A79.269,79.269,0,0,1,435.238,180.5L432.3,203.988a10.685,10.685,0,0,1-10.582,9.344h-34.5a10.667,10.667,0,0,1-10.582-9.344L373.691,180.5a79.579,79.579,0,0,1-17.473-10.391L334.1,179.434a10.669,10.669,0,0,1-13.4-4.5l-17.234-29.867a10.681,10.681,0,0,1,2.793-13.848l19.223-14.547a76.649,76.649,0,0,1-1.023-10.008,80.861,80.861,0,0,1,1-10L306.238,82.109a10.672,10.672,0,0,1-2.8-13.844L320.7,38.4a10.673,10.673,0,0,1,13.379-4.5L356.2,43.242a79.269,79.269,0,0,1,17.473-10.391l2.941-23.488A10.682,10.682,0,0,1,387.2,0h34.5a10.661,10.661,0,0,1,10.582,9.344l2.945,23.488a79.716,79.716,0,0,1,17.473,10.387l22.121-9.32a10.669,10.669,0,0,1,13.4,4.5l17.234,29.867a10.675,10.675,0,0,1-2.793,13.844Zm-79-32.66a42.666,42.666,0,1,0,42.664,42.664A42.655,42.655,0,0,0,404.457,64Zm0,0" transform="translate(0.001 298.668)"/>
-  </g>
+	<g>
+		<g>
```
Comment (NipuniBhagya) [inline modules/theme/src/themes/default/assets/images/icons/gears-icon.svg:24]: Shall we format the SVGs properly?

File: modules/theme/src/themes/dark/assets/images/icons/plug-icon.svg [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/dark/assets/images/icons/plug-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```
Any reason to change it?

File: modules/theme/src/themes/default/assets/images/icons/key-icon.svg [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/icons/key-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```
Shall we use UTF-8 encoding ?

File: modules/theme/src/themes/default/assets/images/icons/pencil-icon.svg [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/icons/pencil-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```
Shall we use UTF-8 encoding ?

File: modules/theme/src/themes/default/assets/images/icons/user-icon.svg [hunk 5]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
+<?xml version="1.0" encoding="utf-8"?>
 
 <!--
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/icons/user-icon.svg:4]:
```suggestion
 * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Don't change license year of old files

File: modules/theme/src/themes/default/assets/images/icons/user-icon.svg [hunk 6]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/icons/user-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```
Any reason to change ?

File: modules/theme/src/themes/default/assets/images/icons/plug-icon.svg [hunk 7]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/icons/plug-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```

File: modules/theme/src/themes/default/assets/images/icons/database-icon.svg [hunk 8]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/icons/database-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```

File: modules/theme/src/themes/default/assets/images/icons/claims-icon.svg [hunk 9]
```diff
@@ -18,24 +18,8 @@
...
-        <path class="path fill primary" d="M17.5,0h357a17.5,17.5,0,0,1,0,35H17.5a17.5,17.5,0,0,1,0-35Z"
-              transform="translate(1857 2236)"/>
-        <path class="path fill primary" d="M17.5,0h357a17.5,17.5,0,0,1,0,35H17.5a17.5,17.5,0,0,1,0-35Z"
-              transform="translate(1857 2171)"/>
-        <path class="path fill primary" d="M19.5,0h173a19.5,19.5,0,0,1,0,39H19.5a19.5,19.5,0,0,1,0-39Z"
-              transform="translate(2037 2100)"/>
-    </g>
+<svg version="1.1" id="claim-icon" class="icon" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+     viewBox="0 0 14 14" style="enable-background:new 0 0 14 14;" xml:space="preserve">
+  <path class="path fill primary claims"
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/icons/claims-icon.svg:23]:
```suggestion
  <path class="path fill primary"
```

File: modules/theme/src/themes/dark/assets/images/icons/pencil-icon.svg [hunk 10]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/dark/assets/images/icons/pencil-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```
Shall we use UTF-8 encoding ?

File: modules/theme/src/themes/dark/assets/images/icons/claims-icon.svg [hunk 11]
```diff
@@ -18,24 +18,8 @@
...
-        <path class="path fill primary" d="M17.5,0h357a17.5,17.5,0,0,1,0,35H17.5a17.5,17.5,0,0,1,0-35Z"
-              transform="translate(1857 2236)"/>
-        <path class="path fill primary" d="M17.5,0h357a17.5,17.5,0,0,1,0,35H17.5a17.5,17.5,0,0,1,0-35Z"
-              transform="translate(1857 2171)"/>
-        <path class="path fill primary" d="M19.5,0h173a19.5,19.5,0,0,1,0,39H19.5a19.5,19.5,0,0,1,0-39Z"
-              transform="translate(2037 2100)"/>
-    </g>
+<svg version="1.1" id="claim-icon" class="icon" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+     viewBox="0 0 14 14" style="enable-background:new 0 0 14 14;" xml:space="preserve">
+  <path class="path fill primary claims"
```
Comment (jeradrutnam) [inline modules/theme/src/themes/dark/assets/images/icons/claims-icon.svg:23]:
```suggestion
  <path class="path fill primary"
```

File: modules/theme/src/themes/dark/assets/images/icons/database-icon.svg [hunk 12]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/dark/assets/images/icons/database-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```

File: modules/theme/src/themes/dark/assets/images/icons/key-icon.svg [hunk 13]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (jeradrutnam) [inline modules/theme/src/themes/dark/assets/images/icons/key-icon.svg:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```

Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/key-icon.svg:1]: Fixed

File: modules/theme/src/themes/dark/assets/images/icons/database-icon.svg (hunk 12 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/database-icon.svg:1]: Fixed

File: modules/theme/src/themes/dark/assets/images/icons/claims-icon.svg (hunk 11 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/claims-icon.svg:23]: Fixed

File: modules/theme/src/themes/dark/assets/images/icons/pencil-icon.svg (hunk 10 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/pencil-icon.svg:1]: Fixed

File: modules/theme/src/themes/default/assets/images/icons/claims-icon.svg (hunk 9 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/default/assets/images/icons/claims-icon.svg:23]: fixed

File: modules/theme/src/themes/dark/assets/images/icons/plug-icon.svg (hunk 2 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/plug-icon.svg:1]: fixed

File: modules/theme/src/themes/default/assets/images/icons/key-icon.svg (hunk 3 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/default/assets/images/icons/key-icon.svg:1]: fixed

File: modules/theme/src/themes/default/assets/images/icons/pencil-icon.svg (hunk 4 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/default/assets/images/icons/pencil-icon.svg:1]: fixed

File: modules/theme/src/themes/default/assets/images/icons/user-icon.svg (hunk 5 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/default/assets/images/icons/user-icon.svg:4]: fixed

File: modules/theme/src/themes/default/assets/images/icons/user-icon.svg (hunk 6 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/default/assets/images/icons/user-icon.svg:1]: fixed

File: modules/theme/src/themes/default/assets/images/icons/plug-icon.svg (hunk 7 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/default/assets/images/icons/plug-icon.svg:1]: fixed

File: modules/theme/src/themes/default/assets/images/icons/database-icon.svg (hunk 8 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/default/assets/images/icons/database-icon.svg:1]: fixed

File: modules/theme/src/themes/default/assets/images/icons/gears-icon.svg (hunk 1 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/default/assets/images/icons/gears-icon.svg]: Done

**Pull request merged**